src/online/atc-info.class.ts

```diff
@@ -0,0 +1,24 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ATCInfo {
+  @ApiProperty({ description: 'The atc callsign', example: 'EBBR_TWR' })
+  callsign:string;
```

Comment:
Spaces missing in multiple locations. Use `npm run lint:fix` to fix and find styling issues.

```suggestion
  callsign: string;
```